apps/emqx/src/emqx_router.erl

```diff
@@ -196,6 +190,26 @@ print_routes(Topic) ->
         match_routes(Topic)
     ).
 
+-spec cleanup_routes(node()) -> ok.
+cleanup_routes(Node) ->
+    Patterns = [
+        #route{_ = '_', dest = Node},
+        #route{_ = '_', dest = {'_', Node}}
+    ],
+    [
+        mnesia:delete_object(?ROUTE_TAB, Route, write)
+     || Pat <- Patterns,
+        Route <- mnesia:match_object(?ROUTE_TAB, Pat, write)
+    ].
```

Comment:
Would it make sense to use a cursor to avoid full instantiation?

Reply:
It absolutely would. Since it's a preexisting piece of code, and I'm currently trying not to increase the PR's scope too much, I think it would be best-suited for a followup PR.